config/libopensplice69_bionic.yaml

```
@@ -0,0 +1,6 @@
+name: libopensplice69_bionic
+method: http://packages.osrfoundation.org/gazebo/ubuntu-prerelease
+suites: [bionic]
+component: main
+architectures: [amd64, arm64, armhf, source]
+filter_formula: Package (= libopensplice69), $Version (% 6.9.190705+osrf1-2~bionic~bionic)
```

```suggestion
filter_formula: Package (= libopensplice69), $Version (% 6.9.190705+osrf1-2*)
```
Combining Xenial and Bionic into a single PR.